public/index.html

```diff
@@ -1,9 +1,10 @@
 <!DOCTYPE html>
-
 <html lang="en">
-	<head>
-		<meta charset="utf-8" />
-		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <title><%= htmlWebpackPlugin.options.title %></title>
```

Comment:
```suggestion
    <title>Stevenson.Space</title>
```
or 
```suggestion
    <title>Stevenson High School</title>
```
Otherwise it shows up as "shs2", and the ```name``` in `package.json` does not permit spaces.

Reply:
oo good catch!